### PR TITLE
[ADD] Create Red October library

### PR DESCRIPTION
**Pull request by lasley**
Python Red October Library
==========================

This library allows you to interact with a remote Red October Instance using Python.

### CODE REVIEW

**Comment by lasley** [comment]
cc @YannickB here's our key encryption I think

File: README.rst [hunk 1]
```diff
@@ -5,6 +5,22 @@ Python Red October Library
...
+
+Authorized persons can delegate their credentials to the server for a period of time.
+The server can decrypt any previously-encrypted payloads as long as the appropriate number
+of people have delegated their credentials to the server.
+
+This architecture allows Red October to act as a convenient decryption service. Other
+systems, including CloudFlare’s build system, can use it for decryption and users can
+delegate their credentials to the server via a simple web interface. All communication
+with Red October is encrypted with TLS, ensuring that passwords are not sent in the clear.
+
```
Comment (lasley) [inline README.rst:22]: Add link to cloudflare blog post

File: README.rst [hunk 2]
```diff
@@ -14,17 +30,19 @@ Setup
 Usage
 -----
 
+* `API Documentation <https://laslabs.github.io/python-red-october>`_
+
 Known Issues / Road Map
 -----------------------
 
 -  Installation, setup, usage - in ReadMe
 -  Add a Certificate Request data structure
```
Comment (lasley) [inline README.rst]: This note was from CFSSL

File: README.rst (hunk 1 above)
Comment (tedsalmon) [reply README.rst:22]: Should we also link to https://github.com/cloudflare/redoctober ?

**Review by tedsalmon — APPROVED** [review]
Made a comment in reply to your review but otherwise it LGTM! 👍

**Comment by lasley** [comment]
Thanks @t3ddftw - comments attended to, I'll squash and merge

**Pull request merged**